content/docs/command-reference/update.md

```diff
@@ -49,11 +49,12 @@ $ dvc update --rev master
   directory and its subdirectories for import stage `.dvc` files to inspect. If
   there are no directories among the targets, this option is ignored.
 
-- `--to-remote` - update a `.dvc` file created with `dvc import-url` and
-  [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
-  the new data directly to remote storage (the default one unless `-r` is used).
-  No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the
-  data locally. This option can't be used with DVC or Git repository imports.
+- `--to-remote` - update a `.dvc` file created with `dvc import-url` and store
+  the latest data directly
+  [on remote storage](/doc/command-reference/import-url#straight-to-remote) (the
+  default one unless `-r` is specified). No changes are done in the
```

Comment:
`No changes are done` contradicts to the `update a `.dvc` I think

Reply:
Changed to "Tracked data is not changed".